[2024-12-26T14:39:33.644+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: etl_pipeline.run_currency_exchange_etl manual__2024-12-26T14:39:32.359900+00:00 [queued]>
[2024-12-26T14:39:33.647+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: etl_pipeline.run_currency_exchange_etl manual__2024-12-26T14:39:32.359900+00:00 [queued]>
[2024-12-26T14:39:33.647+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-12-26T14:39:33.648+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2024-12-26T14:39:33.648+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-12-26T14:39:33.651+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): run_currency_exchange_etl> on 2024-12-26 14:39:32.359900+00:00
[2024-12-26T14:39:33.653+0000] {standard_task_runner.py:55} INFO - Started process 49 to run task
[2024-12-26T14:39:33.655+0000] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'etl_pipeline', 'run_currency_exchange_etl', 'manual__2024-12-26T14:39:32.359900+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/currency_exchange_dag.py', '--cfg-path', '/tmp/tmppgy14n7c']
[2024-12-26T14:39:33.655+0000] {standard_task_runner.py:83} INFO - Job 17: Subtask run_currency_exchange_etl
[2024-12-26T14:39:33.662+0000] {warnings.py:109} WARNING - /home/airflow/.local/lib/python3.9/site-packages/airflow/settings.py:249: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-12-26T14:39:33.684+0000] {task_command.py:388} INFO - Running <TaskInstance: etl_pipeline.run_currency_exchange_etl manual__2024-12-26T14:39:32.359900+00:00 [running]> on host b10b41203f98
[2024-12-26T14:39:33.717+0000] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=etl_pipeline
AIRFLOW_CTX_TASK_ID=run_currency_exchange_etl
AIRFLOW_CTX_EXECUTION_DATE=2024-12-26T14:39:32.359900+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-12-26T14:39:32.359900+00:00
[2024-12-26T14:39:33.719+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-12-26T14:39:33.719+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/airflow/dags/currency_exchange_etl.py']
[2024-12-26T14:39:33.723+0000] {subprocess.py:86} INFO - Output:
[2024-12-26T14:39:34.100+0000] {subprocess.py:93} INFO - USD INR 85.234 2024-12-26 UTC 85.2333 85.238
[2024-12-26T14:39:34.100+0000] {subprocess.py:93} INFO - Error connecting to the database: connection to server on socket "/var/run/postgresql/.s.PGSQL.5432" failed: No such file or directory
[2024-12-26T14:39:34.106+0000] {subprocess.py:93} INFO - 	Is the server running locally and accepting connections on that socket?
[2024-12-26T14:39:34.108+0000] {subprocess.py:93} INFO - 
[2024-12-26T14:39:34.114+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-12-26T14:39:34.133+0000] {taskinstance.py:1318} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=run_currency_exchange_etl, execution_date=20241226T143932, start_date=20241226T143933, end_date=20241226T143934
[2024-12-26T14:39:34.158+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2024-12-26T14:39:34.167+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
